### DISCUSSION TITLE
Vertical spacing/alignments for some fonts

### DISCUSSION THREAD

**Discussion by marlun** [kovidgoyal/kitty#3462] [Q&A] [answered]
I'm trying the font Codelia (https://tosche.net/fonts/codelia) and for some reason it's aligned more to the top than the bottom as you can see in the picture. Is this a font issue or an issue with Kitty? Can I modify somehow?

![image](https://user-images.githubusercontent.com/19663/114172291-c1487600-9935-11eb-8d94-322c37fce242.png)

**Comment by kovidgoyal** [comment, marked as answer]
alignment is set by the baseline in the font. You can possibly workaround it a bit by using adjust_cell_height but in general it needs to be fixed in the font.

**Reply by marlun** [reply]
Thanks I've contacted the font author as well. The following "fixed" it for me:

    adjust_line_height  -5

![image](https://user-images.githubusercontent.com/19663/114174778-687adc80-9939-11eb-8f33-cb71d6a58daa.png)

**Reply by karimknaebel** [reply]
Is it possible to `adjust_line_height` for the fonts used in `symbol_map`? My main font works fine, but some of the fonts I use in `symbol_map` don't seem to have the correct line_height.
